### ISSUE TITLE
Removing media blanks?

### ISSUE THREAD

**Issue by antiburnsey** [mzmine/mzmine2#357]
Hi,

I'm using MZmine for processing my untargeted metabolomics data (Waters files, so already had trouble with multiple scan functions appearing on the same chromatogram).  Currently I'm removing the media components from my samples by manually stripping the blank from each individual file in Masslynx, which is a bit time consuming.  Can mzmine do this e.g. by subtracting the blank's peak list from the sample peak list?  

Thanks!

Josh

**Comment by tomas-pluskal** [comment]
Hi Josh,
There is no such function specifically in MZmine, but you can achieve that by aligning all your peaklists (including the blanks) and then removing those peaks that appear in your blank samples above some intensity threshold.

**Comment by antiburnsey** [comment]
Hi Tomas,

Thanks for the answer.  I've removed the blank peaks after aligning by selecting status -> detected in blank -> highlight and delete selected rows, which hopefully should just leave the metabolites.  Is it possible to go from peak list back to chromatogram and show the metabolite-only list?  If not, do you know of any other free software that does do blank removal and show the resulting chromatogram?

Thanks again.

**Comment by MuyMal** [comment]
Hi Josh (and Tomas),

Thank you for posting this question, as it was one I had as well. After you remove the peaks that appear in your blanks, are you able to visualize a scatter plot of the resulting peak list? I seem to be getting an error:

"Exception in thread "AWT-EventQueue-0" java.lang.IllegalArgumentException: A positive range length is required: Range[8.988, 8.988]"

Many thanks,

Mallory

**Comment by antiburnsey** [comment]
Hi Mallory,

I gave the scatter peak plot a try - seemed to work for me!  Though it wasn't a very interesting scatter plot since all the peaks are only present in the one sample  Are you doing everything the same (align, right click delete, visualise)?

Also - I found that highlighting all the remaining peaks after deleting -> XIC (dialog) -> select only the sample raw data file seems to work nicely for my original question, so I think that's sorted.
